src/home/DappsCarousel.tsx

```diff
@@ -158,9 +151,20 @@ function DappsCarousel({ onSelectDapp }: Props) {
               <Text style={styles.dappName} numberOfLines={1} ellipsizeMode="tail">
                 {dapp.name}
               </Text>
-            </>
+            </Card>
           </Touchable>
         ))}
+
+        <Touchable onPress={onPressAllDapps}>
+          <Card style={styles.viewAllCard} rounded={true} shadow={Shadow.SoftLight}>
+            <View style={[styles.icon, styles.viewAllIcon]}>
+              <ArrowRight />
+            </View>
+            <Text style={styles.dappName} numberOfLines={1} ellipsizeMode="tail">
+              {t('allDapps')}
+            </Text>
+          </Card>
+        </Touchable>
```

Comment:
nit: maybe all items should be the same component with different data?

Reply:
to get this guy to print within the dapps loop, i'd have to make up some data to get this into the `Dapp` interface (many redundant properties) / make the `onPress` responsive to the dapp "type" / add a conditional styling on the image since this one has a green background - i could also export the image with the green background but it makes the icon less reusable. with the implementation here, i also could easily use a different test id here (i.e. none, lol) so the tests feel more correct. i did think about this suggestion before but i don't think it is as readable as this, what do you think?

Reply:
Yes that's fine, we just need to remember to update both places.
Then we need the Touchable inside the Card here too.

Reply:
oops 🙈